DataAccess/DataAccess.SQLite/LiteTable.hpp

```diff
@@ -1,34 +1,36 @@
 #pragma once
+#include <DataAccess/QueryBuilder.hpp>
+
 #include "LiteAdapter.hpp"
-#include "QueryBuilder.hpp"
 
 namespace DataAccess
 {
-/** @class SQLite table.
- *   @brief SQLite table class.
- *   @details You can see some examples below for how to use it.
- *   @code
- *    LiteTable("tableName1").Select()->columns({"column1", "column2",
- * ...})->where("condition")->...->execute()/OR/getQuery(); LiteTable("tableName2").Insert()->field(1,
- * "a")->field(...)->...->returning({"column1", "column2", ...})->execute()/OR/getQuery();
- *    LiteTable("tableName3").Update()->fields(pair{"column1", 1}, pair{"column2",
- * "strData"})->where("condition")->...->execute()/OR/getQuery();
- *    LiteTable("tableName4").Delete()->where("condition")->...->execute()/OR/getQuery();
- *   @endcode
+/**
+ * @class LiteTable
+ * @brief SQLite table class.
+ * @details You can see some examples below for how to use it.
+ * @code
+ *  LiteTable("tableName1").Select()->columns({"column1", "column2", ...})->where("condition")
+ *      ->...->execute()/OR/getQuery(); LiteTable("tableName2").Insert()
+ *      ->field(1, "a")->field(...)->...->returning({"column1", "column2", ...})->execute()/OR/getQuery();
+ *  LiteTable("tableName3").Update()->fields(pair{"column1", 1}, pair{"column2",
+ *      "strData"})->where("condition")->...->execute()/OR/getQuery();
+ *  LiteTable("tableName4").Delete()->where("condition")->...->execute()/OR/getQuery();
+ * @endcode
  */
 class LiteTable : public QueryBuilder<QSqlQuery>
 {
 public:
-    LiteTable(const std::string& tableName)
-        : QueryBuilder(DatabaseType::DB_LITE, tableName) 
+    /// Table constructor
+    explicit LiteTable(const std::string& tableName) : QueryBuilder(Utility::DatabaseType::DB_LITE, tableName)
     {
-        _adapter = LiteAdapter::getInstance<LiteAdapter>();
+        _adapter = LiteAdapter::Instance("local.db");
```

Comment:
Method Instance is using only inside adapter. You have to get instance from getInstance() method.

Reply:
okay, but `LiteAdapter::getInstance<LiteAdapter>();`{.cpp} doesn't look good